spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/data/redis/RedisAutoConfigurationTests.java

```diff
@@ -165,6 +166,8 @@ public void testRedisConfigurationWithPool() {
 					assertThat(poolConfig.getMaxIdle()).isEqualTo(4);
 					assertThat(poolConfig.getMaxTotal()).isEqualTo(16);
 					assertThat(poolConfig.getMaxWaitMillis()).isEqualTo(2000);
+					assertThat(poolConfig.getTimeBetweenEvictionRunsMillis())
```

Comment:
Would you like to make an equivalent change to the tests for configuring Jedis as well?

Reply:
Same discussion as [comments](https://github.com/spring-projects/spring-boot/pull/16703#discussion_r280812432)

Reply:
I made an equivalent change to the tests for configuring Jedis.